### ISSUE TITLE
Update auf 0.9.5

### ISSUE THREAD

**Issue by robinlefleur** [JsBergbau/BaikalAnleitung#25]
Wenn man von einer älteren Baikal-Version (z.B. 0.9.1) auf eine neuere Version (z.B. 0.9.5) aktualisieren möchte und noch eine veraltete PHP-Version (7.3) verwendet, ist es erforderlich, auch PHP auf eine neuere Version zu aktualisieren.

Da es keinen offiziellen Support für PHP 8.1 auf dem Raspberry Pi Zero gibt, habe ich für Baikal auf einen neueren Raspberry Pi Zero 2 W gewechselt. Für die Aktualisierung auf PHP 8.3 habe ich [dieses Tutorial](https://www.sascha-syring.net/webserver-installieren/php-installieren/alternative-php-version-8-1-installieren/) verwendet.

Nach der erfolgreichen PHP-Aktualisierung muss noch eine Anpassung in der Konfigurationsdatei baikal-nginx.site vorgenommen werden. Die Zeile `fastcgi_pass unix:/run/php/php7.3-fpm.sock;` muss entsprechend der installierten PHP-Version geändert werden, in diesem Fall zu `fastcgi_pass unix:/run/php/php8.3-fpm.sock;`

Nach diesen Schritten sollte die neuste Baikal-Version erfolgreich installiert sein.

**Comment by JsBergbau** [comment]
Hallo,

danke für die Anleitung.

PHP von Drittanbieterquellen zu installieren rate ich dringend ab. Wenn es Sicherheitslücken gibt, muss man sich darauf verlassen, dass dieser Anbieter diese ebenfalls behebt und dies möglichst schnell. Die Pakete von den Distributionen werden durch Debian immer auf dem aktuellen Sicherheitsstandard gehalten. 

Laut den Releasenotes https://github.com/sabre-io/Baikal/releases wird PHP 7.2 und aufwärts unterstützt. Mit PHP 8.2 scheint es hingegeben immer noch ein paar Probleme zu geben. 

Es scheint demnach keine Notwendigkeit zu geben PHP auf Version 8 zu bringen.

**Comment by robinlefleur** [comment]
Ja, die Release Notes haben mich auch verwirrt, ich hab jedoch diesen Fehler dann gehabt: `Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". `

**Comment by JsBergbau** [comment]
Hast du eventuell weitere Paketquellen von Drittanbietern? Composer kommt ja über `apt` und sollte zur mitgelieferten Version passen.

**Comment by robinlefleur** [comment]
Ich hatte einen ganz frischen Raspberry Pi Zero gehabt und dann nur das Tutorial befolgt.
